### ISSUE TITLE
Problem: In pointer file, ID refs are AIP names instead of UUIDs with non-numeric prefixes

### ISSUE THREAD

**Issue by evelynPM** [archivematica/Issues#660]
**Expected behaviour**
ID refs should have a non-numeric prefix to conform to NCName requirements.  For example, in the AIP METS file an ID ref in the fileSec is "file-UUID", like this: 

'<mets:file GROUPID="Group-869d9b1c-0a7b-4061-8832-75430086015b" ID="file-869d9b1c-0a7b-4061-8832-75430086015b" ADMID="amdSec_1">' 

**Current behaviour**
In the AIP pointer file the ID refs are just the AIP names with UUIDs appended, like this: 

'<mets:file ID="MyAIP-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5" GROUPID="Group-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5" ADMID="amdSec_733264">'

In this case the pointer file will still validate because the AIP name happens to start with letters instead of numbers. However, if the AIP name starts with a number, the pointer file will fail validation. Here is some sample validation output for an AIP named "1920-2":
-line 166, column 142: cvc-datatype-valid.1.2.1: '1920-2-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5' is not a valid value for 'NCName'.
-line 166, column 142: cvc-attribute.3: The value '1920-2-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5' of attribute 'ID' on element 'mets:file' is not valid with respect to its type, 'ID'.
-line 174, column 72: cvc-datatype-valid.1.2.1: '1920-2-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5' is not a valid value for 'NCName'.
-line 174, column 72: cvc-attribute.3: The value '1920-2-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5' of attribute 'FILEID' on element 'mets:fptr' is not valid with respect to its type, 'IDREF'.

**Steps to reproduce**
Create an AIP with a number for the AIP name and validate the pointer file.

**Your environment (version of Archivematica, OS version, etc)**
AM 1.9.1 sandbox

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by ross-spencer** [comment]
Related, possible duplicate of: https://github.com/archivematica/Issues/issues/380

**Comment by evelynPM** [comment]
As a solution I would suggest replacing the AIP name in ID refs with `file-`, just like we do for AIP METS files. So instead of `'<mets:file ID="MyAIP-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5" GROUPID="Group-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5" ADMID="amdSec_733264">'`we should have  `'<mets:file ID=file-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5" GROUPID="Group-23e81b05-db90-4f0d-bfc3-96eb65dc4ba5" ADMID="amdSec_733264">'`. This is a reasonable solution because the AIP is being described as a single file in the pointer file (i.e. a zipped package).

**Comment by ross-spencer** [comment]
@joel-simpson @sromkey this seemed to escape the `1.10` milestone Monday but believe the result was to be added.

**Comment by ross-spencer** [comment]
Having analysed this in a little more detail, the change needs to be made to both the Storage Service and mets-reader-writer. 

In the storage service, the `filegrp` is generated with [this instantiation of mets-reader-writer](https://github.com/artefactual/archivematica-storage-service/blob/946a513da208f96c87b00f0ae86ad011d7e9baac/storage_service/locations/models/package.py#L1287-L1290).

Which refers to this [init method](https://github.com/artefactual-labs/mets-reader-writer/blob/d95939cabdfdc25cb1bf67df0c84bd0d6e6a73ff/metsrw/fsentry.py#L112-L125). And then the derivation for the file id happens [here](https://github.com/artefactual-labs/mets-reader-writer/blob/d95939cabdfdc25cb1bf67df0c84bd0d6e6a73ff/metsrw/fsentry.py#L198-L208).

The mets-reader-writer constructor is getting a little busy though, so this (as a design pattern) might need to be discussed further/revisited again in future.

**Comment by ross-spencer** [comment]
Requires: https://github.com/archivematica/Issues/issues/732
